src/webview_provider/newLineagePanel.ts

```diff
@@ -286,6 +291,20 @@ export class NewLineagePanel implements LineagePanelView {
     );
   }
 
+  private sendExternalProjectTelemetryEventSent(body: {
```

Comment:
Sends an event only once per session

Reply:
I personally prefer to send it as an attribute of our events and base it on if dbt-loom is installed or not (that means the intention is that the user would like to use it).

@anandgupta42 : WDYT?

Reply:
Yes. We should only send it if there project has dbt loom configured.